CONTRIBUTING.md

```diff
@@ -64,3 +64,55 @@ Other Tips
     your existing commit message by running:
 
         $ git commit --amend
+
+Checklist for Contributed Code
+------------------------------
+
+A pull request for a new feature will be reviewed to see if it meets the following requirements.  For any pull request, an astropy maintainer can help to make sure that the pull request meets the requirements for inclusion in the package.  
+
+**Scientific Quality**
+  * Is the submission relevant to astronomy? 
+  * Are references included to the origin source for the algorithm?
+  * Does the code perform as expected?
+  * Has the code been tested against previously existing implimentations?
+
+**Code Quality**
+  * Are the [coding guidelines](http://docs.astropy.org/en/latest/development/codeguide.html)
+    followed?
+  * Is the code compatible with Python 2.6, 2.7, as well as 3?
```

Comment:
2.6, 2.7, and >= 3.3.  I think we dropped support for Python 3.3 > v >= 3.0.  For the most part also that's caught by the build bots.  But it is something I often manually inspect too (since not all code is covered by tests, and sometimes the builds are slow to run :)